WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayInformationViewController.swift

```diff
@@ -129,6 +135,7 @@ struct SetUpTapToPayInformationView: View {
             .scrollVerticallyIfNeeded()
         }
         .padding()
+        .onAppear(perform: viewModel.viewDidAppear)
```

Comment:
Just to clarify if I understood correctly why this method call: This is basically a configuration method, right? We're not adding it to the viewmodel's init because we don't want to rely on the viewModel's lifecycle, and wait for this to be initialized to trigger this configuration. And at the same time, we want the config logic to be testable, so we extract it from the view to the model.

Reply:
IIRC, we're not using the init because the view model is created optimistically, the user may not actually open the screen. We use this `viewModel.viewDidAppear` to trigger the notification, which we respond to in other places by removing all the badges... so we only want to do that when it actually appears.